openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/components/SelectTestSuite.tsx

```diff
@@ -49,14 +49,14 @@ const SelectTestSuite: React.FC<SelectTestSuiteProps> = ({
   const { isAdminUser } = useAuth();
   const { isAuthDisabled } = useAuthContext();
   const [form] = useForm();
-  const hasAccess = isAdminUser || isAuthDisabled;
+  const hasAccess = isAdminUser ?? isAuthDisabled;
```

Comment:
here we need OR operation, access will be given if its admin or auth is disable, with use of ?? it will only fall back to authDisable when isAdminUser is undefined or null